HaulageApplication/Haulage/BaseClasses/Accounting/Driver.cs

```diff
@@ -6,28 +6,23 @@
 
 namespace Haulage.BaseClasses.Accounting
 {
-    public class Driver : Person
+    public class Driver : User
     {
-        enum qualifications
-        {
-            fragileAllowed,
-            qualification2,
-            qualification3,
-        }
-        public void recordExpenses()
-        {
+        public int UserId { get; set; }
```

Comment:
most of these properties should be on the User class as they are all transferable to customer.cs and administrator.cs when getting from the user table.

Reply:
moved to User